dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer/MergeCartsTest.php

```diff
@@ -192,9 +192,6 @@ public function testMergeCartsWithEmptySourceCartId()
      */
     public function testMergeCartsWithEmptyDestinationCartId()
```

Comment:
I would ask you to cover this case with an additional test. The test will assert that without providing the destination cart id, the current cart is being used.

Reply:
Hello @rogyar 
Thank you for your review.
I extended the tests.
Please check.

Thank you in advance.